jobs/config.json

```diff
@@ -6781,6 +6781,22 @@
       "UNKNOWN"
     ]
   },
+  "pull-kops-e2e-kubernetes-aws-scenario": {
+    "args": [
+      "--extract=release/stable",
+      "--build-kops",
+      "--cluster=",
```

Comment:
Yeah I'm pretty sure we had logic somewhere to auto-select the cluster if undefined? We should copy that over.